bundles/org.openhab.binding.haassohnpelletoven/src/main/resources/OH-INF/i18n/haassohnpelletoven_xx_XX.properties

```diff
@@ -0,0 +1,29 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
+# FIXME: please do not add the file to the repo if you add or change no content
```

Comment:
You need to replace the xx in the filename by the locale you want to cover.

As the english locale is already covered by the XML file, this file could be removed.